q2studio/api/formats.py

```diff
@@ -16,5 +16,5 @@
 
 @formats.route('/importable', methods=['POST'])
 def get_importable_formats():
-    ret = [t.format.__name__ for t in PLUGIN_MANAGER.importable_formats]
```

Comment:
Minor: you don't need to call `.keys()` when iterating over a `dict`. This code can be simplified to:

```python
ret = list(PLUGIN_MANAGER.importable_formats)
```